tools/pythonpkg/duckdb_python.cpp

```diff
@@ -183,7 +183,10 @@ static void InitializeConnectionMethods(py::module_ &m) {
 	    .def("unregister_filesystem", &PyConnectionWrapper::UnregisterFilesystem, "Unregister a filesystem",
 	         py::arg("name"), py::arg("connection") = py::none())
 	    .def("list_filesystems", &PyConnectionWrapper::ListFilesystems,
-	         "List registered filesystems, including builtin ones", py::arg("connection") = py::none());
+	         "List registered filesystems, including builtin ones", py::arg("connection") = py::none())
+	    .def("filesystem_is_registered", &PyConnectionWrapper::FileSystemIsRegistered,
```

Comment:
What's the rationale behind adding this method, instead of just using `'name' in list_filesystems()`?

Reply:
Just wanted a efficient way to check, the existing route returned a py::list - so no real reason, but it also doesn't hurt anyone, right?